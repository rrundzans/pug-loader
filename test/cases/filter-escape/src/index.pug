//- filter with block syntax
pre: code.language-html
  :escape
    <!-- HTML -->
    <div class="content">
      <p>&copy; M&M'S</p>
    </div>

//- inline syntax
p
  :escape The <strong> element has the closing </strong> tag.

p.
  The #[:escape <html>] element is the root element.<br>
  Inside the #[:escape <html>] element there is a #[:escape <body>] element.