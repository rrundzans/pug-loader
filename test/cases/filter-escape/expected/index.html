<pre><code class="language-html">&lt;!-- HTML --&gt;
&lt;div class=&quot;content&quot;&gt;
  &lt;p&gt;&amp;copy; M&amp;M'S&lt;/p&gt;
&lt;/div&gt;
</code></pre><p>The &lt;strong&gt; element has the closing &lt;/strong&gt; tag.</p><p>The &lt;html&gt; element is the root element.<br>
Inside the &lt;html&gt; element there is a &lt;body&gt; element.</p>